Settings-default.toml

```diff
@@ -143,7 +148,7 @@ user_registration = true
 quota_check = false
 # default_available_quota = 999999
 admin_email = "admin@localhost"
-admin_password = "admin" # Initial password for the user "admin", which is created on startup
+admin_password = "adminadmin" # Initial password for the user "admin", which is created on startup
```

Comment:
Much more secure now :wink:

Reply:
otherwise it doesn't get through the user input validation